das/daser_test.go

```diff
@@ -17,7 +17,8 @@ import (
 
 	"github.com/celestiaorg/celestia-node/fraud"
 	"github.com/celestiaorg/celestia-node/header"
-	"github.com/celestiaorg/celestia-node/service/share"
+	"github.com/celestiaorg/celestia-node/nodebuilder/share"
+	service "github.com/celestiaorg/celestia-node/share"
```

Comment:
This should be called share not service.

Reply:
done